frontend/lib/src/components/widgets/ButtonGroup/ButtonGroup.tsx

```diff
@@ -294,4 +333,4 @@ function ButtonGroup(props: Readonly<Props>): ReactElement {
   )
 }
 
-export default ButtonGroup
+export default memo(ButtonGroup)
```

Comment:
Note: this makes a big difference in the amount this component gets rendered. This reduced the render occurrence from > 20 to ~2

Reply:
Awesome 🥳 can you add a tech debt entry to update this for all our elements and maybe one to investigate why we even have these 20 rerenders from outside (this seems a bit high :( )

Reply:
Good idea, I have added an item 👍